xrmocap/model/architecture/multi_view_pose_transformer.py

```diff
@@ -51,12 +52,12 @@ def __init__(self, is_train: bool, logger: Union[None, str,
                  use_feat_level: list, n_cameras: int, query_embed_type: str,
                  with_pose_refine: bool, loss_weight_loss_ce: float,
                  loss_per_kp: float, aux_loss: bool, pred_conf_threshold: list,
-                 pred_class_fuse: str, projattn_pose_embed_mode: str,
+                 pred_class_fuse: str, projattn_pos_embed_mode: str,
                  query_adaptation: bool, convert_kp_format_indexes: list,
-                 backbone_setup: dict, decoder_layer_setup: dict,
-                 decoder_setup: dict, pos_encoding_setup: dict,
-                 pose_embed_setup: dict, matcher_setup: dict,
-                 criterion_setup: dict, space_size: list,
+                 backbone_setup: dict, proj_attn_setup: dict,
+                 decoder_layer_setup: dict, decoder_setup: dict,
+                 pos_encoding_setup: dict, pose_embed_setup: dict,
```

Comment:
I want to make sure, should it be `pose_embed_setup` or `pos_embed_setup`?

Reply:
`pose_embed` is the embedding layers for keypoints.
`pos_embed` is the positional embedding mode for the transformer.